doc/sphinx/addendum/parallel-proof-processing.rst

```diff
@@ -55,6 +57,33 @@ proof does not begin with :cmd:`Proof using`, the system records in an
 auxiliary file, produced along with the ``.vo`` file, the list of section
 variables used.
 
+If a theorem has an incorrect annotation that omits a needed variable, you may see
+a message like this:
+
+   .. code-block::
+
+      File "./Pff.v", line 2372, characters 0-4:
+      Error: The following section variable is used but not declared:
+      precisionNotZero.
+
+      You can either update your proof to not depend on precisionNotZero, or you can
+      update your Proof line from
```

Comment:
```suggestion
      update your ``Proof`` line from
```

Reply:
This is literal text for an error message, so I think no change here.